portfolio/src/main/webapp/graphs.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Graphs</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+        <script type="text/javascript">

```

Comment:
The indentation here seems a little off. I think we should move it a little to the left.

Reply:
Fixed indentation.